### PR TITLE
add plane pre-arms to table

### PR DESCRIPTION
**Pull request by Hwurzburg**
a few copter nitpicks fixed also

### CODE REVIEW

File: common/source/docs/common-prearm-safety-checks.rst [hunk 1]
```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
+[site wiki="plane"]
+
+.. table:: Pre-Arm Failure Messages (Plane Only)
+    :widths: auto
 
+    ======================================================= =================================================== ====================================================
+    Message                                                 Cause                                               Solution
+    ======================================================= =================================================== ====================================================
+    ADSB threat detected                                    ADSB failsafe.  Manned vehicles nearby              See :ref:`ADSB configuration<common-ads-b-receiver>`
+    AHRS not healthy                                        AHRS/EKF is not yet ready                           Wait.  Reboot autopilotFS_LONG_TIMEOUT < FS_SHORT_TIMEOUT
```
Comment (IamPete1) [inline common/source/docs/common-prearm-safety-checks.rst:276]:
```suggestion
    AHRS not healthy                                        AHRS/EKF is not yet ready                           Wait.  Reboot autopilot
```

File: common/source/docs/common-prearm-safety-checks.rst [hunk 2]
```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
+    Invalid THR_FS_VALUE for reversed throttle input        THR_FS_VALUE pwm is not ABOVE the max throttle      Set :ref:`THR_FS_VALUE<THR_FS_VALUE>` above throttle maximum pwm
+    LIM_ROLL_CD too small x                                 Parameter set under 3 degrees                       Increase, 45 deg recommended minimum for adequate control
+    LIM_PITCH_MAX too small x                               Parameter set under 3 degrees                       Increase, 45 deg recommended minimum for adequate control
+    LIM_PITCH_MIN too large x                               Parameter set over 3 degrees                        Increase, 45 deg recommended minimum for adequate control
+    Mode not armable                                        Cannot arm from this mode                           Change Mode
+    Mode not QMODE                                          Q_OPTION set to prevent arming except in QMODE/AUTO Change Mode or reset :ref:`Q_OPTIONS<Q_OPTIONS>` bit 18
+    Motors: Check frame class and type                      Unknown or misconfigured frame class or type        Enter valid frame class and/or type
```
Comment (IamPete1) [inline common/source/docs/common-prearm-safety-checks.rst:288]:
```suggestion
    Motors: Check frame class and type                      Unknown or misconfigured frame class or type        Enter valid frame class (:ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>`) and/or type (:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>`)
```

File: common/source/docs/common-prearm-safety-checks.rst [hunk 3]
```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
+    Motors: Check MOT_PWM_MIN and MOT_PWM_MAX               MOT_PWM_MIN or MOT_PWM_MAX misconfigured            Set :ref:`MOT_PWM_MIN<MOT_PWM_MIN>` = 1000 and :ref:`MOT_PWM_MAX<MOT_PWM_MAX>` = 2000 and repeat the :ref:`ESC calibration <esc-calibration>`
```
Comment (IamPete1) [inline common/source/docs/common-prearm-safety-checks.rst:289]:
```suggestion
    Motors: Check Q_M_PWM_MIN and Q_M_PWM_MAX               Q_M_PWM_MIN or Q_M_PWM_MAX     
 misconfigured            Set :ref:`Q_M_PWM_MIN<Q_M_PWM_MIN>` = 1000 and :ref:`Q_M_PWM_MAX    <Q_M_PWM_MAX>` = 2000 and repeat the :ref:`ESC calibration <esc-calibration>`
```

File: common/source/docs/common-prearm-safety-checks.rst [hunk 4]
```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
+    Motors: MOT_SPIN_ARM > MOT_SPIN_MIN                     MOT_SPIN_ARM is too high or MOT_SPIN_MIN is too low Reducse :ref:`MOT_SPIN_ARM<MOT_SPIN_ARM>` to below :ref:`MOT_SPIN_MIN<MOT_SPIN_MIN>`.  See :ref:`Seting motor range<set-motor-range>`
+    Motors: MOT_SPIN_MIN too high x > 0.3                   MOT_SPIN_MIN parameter value is too high            Reduce :ref:`MOT_SPIN_MIN<MOT_SPIN_MIN>` to below 0.3.  See :ref:`Setting motor range<set-motor-range>`
```
Comment (IamPete1) [inline common/source/docs/common-prearm-safety-checks.rst:291]: `Q_M` not `MOT`

File: common/source/docs/common-prearm-safety-checks.rst [hunk 5]
```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
+    Motors: no SERVOx_FUNCTION set to MotorX                At least one motor output has not been configured   Check SERVOx_FUNCTION values for "Motor1", "Motor2", etc.  Check :ref:`ESC and motor configuration <connect-escs-and-motors>`
+    Q_ASSIST_SPEED is not set                               Q_ASSIST_SPEED has not been set                     Set :ref:`Q_ASSIST_SPEED<Q_ASSIST_SPEED>` = -1 for no Assistance, See :ref:`assisted_fixed_wing_flight`
```
Comment (IamPete1) [inline common/source/docs/common-prearm-safety-checks.rst:293]: Should advise people to set it correctly rather than using the magic -1 to skip the check.

Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:293]: for me -1 is correct, but I can just refer to the page (which now has a note about disabling QASSIT)

File: common/source/docs/common-prearm-safety-checks.rst (hunk 4 above)
Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:291]: yep...copy paste error

File: common/source/docs/common-prearm-safety-checks.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:289]: copy paste from copter error...thanks

File: common/source/docs/common-prearm-safety-checks.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:288]: fine with me

File: common/source/docs/common-prearm-safety-checks.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:276]: oops....have to fix that....alphabetizing copy paste error

File: common/source/docs/common-prearm-safety-checks.rst [hunk 6]
```diff
@@ -42,13 +42,13 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
...
     AHRS: waiting for home                                  GPS has not gotten a fix                            If indoors, go outside.  Ensure compass and accelerometer calibrations have been completed.  Eliminate radio-frequency sources that could interfere with the GPS
     Airspeed 1 not healthy                                  Autopilot is unable to retrieve data from sensor    Check the physical connection and :ref:`configuration <airspeed>`
     AP_Relay not available                                  Parachute misconfigured                             Parachute is controlled via Relay but the relay feature is missing. `Custom build server <https://custom.ardupilot.org/>`__ was probably used, rebuild with relay enabled
-    Auxiliary authorisation refused                         External system has disallowed aurhorisation        Check external authorisation system
+    Auxiliary authorization refused                         External system has disallowed authorization        Check external authorisation system
```
Comment (rmackay9) [inline common/source/docs/common-prearm-safety-checks.rst:51]: Well, this is just american spelling vs UK spelling.. I vote for leaving it with an "s".

**Comment by rmackay9** [comment]
Thanks for this.  one objections but otherwise the copter changes look good and thanks for finding one I missed!

File: common/source/docs/common-prearm-safety-checks.rst (hunk 6 above)
Comment (Hwurzburg) [reply common/source/docs/common-prearm-safety-checks.rst:51]: okay...found some more errors...but not UK vs US

**Pull request merged**